### PR TITLE
Add Universal Profiling to the Hosts docs

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
Closes Issue #3481. 

Adds the Universal Profiling tab to the **Infrastructure > Host** docs.

### CODE REVIEW

File: docs/en/observability/explore-logs.asciidoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ image::images/log-explorer.png[Screen capture of the Logs Explorer]
 == Find your logs
 
 By default, Logs Explorer shows all of your logs.
-If you need to focus on logs from a specific integrations, select the integration from the logs menu:
+If you need to focus on logs from a specific integration, select the integration from the logs menu:
```
Comment (mdbirnstiehl) [inline docs/en/observability/explore-logs.asciidoc:20]: Can ignore this line, just fixing a typo without open a specific PR.

File: docs/en/observability/monitor-infra/host-details-partial.asciidoc [hunk 2]
```diff
@@ -52,18 +52,39 @@ along with the total number of processes in these various states:
...
+[%collapsible]
+.*Universal Profiling*
+====
+
+[role="screenshot"]
+image::images/universal-profiling-overlay.png[Host Universal Profiling]
+
+The *Universal Profiling* tab shows CPU usage down to the application code level.
+From here, you can find the sources of resource usage, and identify code that can be optimized to reduce infrastructure costs.
+The Universal Profile tab has the following views.
```
Comment (dedemorton) [inline docs/en/observability/monitor-infra/host-details-partial.asciidoc:78]:
```suggestion
The Universal Profiling tab has the following views.
```

File: docs/en/observability/monitor-infra/host-details-partial.asciidoc [hunk 3]
```diff
@@ -52,18 +52,39 @@ along with the total number of processes in these various states:
...
+
+|===
+
+| *Flamegraph* | A visual representation of the functions that consume the most resources. Each rectangle represents a function. The rectangle width represents the time spent in the function. The number of stacked rectangles represents the stack depth, or the number of functions called to reach the current function.
+
+| *Top 10 Functions* | A list of the most expensive lines of code on your host. See the most frequently sampled functions, broken down by CPU time, annualized CO2, and annualized cost estimates.
```
Comment (dedemorton) [inline docs/en/observability/monitor-infra/host-details-partial.asciidoc:84]: I don't know what annualized CO2 means. Will users? I just know it's not good for the planet. :-P

**Review by dedemorton — COMMENTED** [review]
Couple of minor comments. Otherwise looks copacetic.

File: docs/en/observability/monitor-infra/host-details-partial.asciidoc [hunk 4]
```diff
@@ -52,18 +52,39 @@ along with the total number of processes in these various states:
...
+[%collapsible]
+.*Universal Profiling*
+====
+
+[role="screenshot"]
+image::images/universal-profiling-overlay.png[Host Universal Profiling]
+
+The *Universal Profiling* tab shows CPU usage down to the application code level.
+From here, you can find the sources of resource usage, and identify code that can be optimized to reduce infrastructure costs.
+The Universal Profiling tab has the following views.
```
Comment (mykolaharmash) [inline docs/en/observability/monitor-infra/host-details-partial.asciidoc:78]: Small suggestion, we might want to also point people to [the full Profiling documentation](https://www.elastic.co/guide/en/observability/current/universal-profiling.html).

**Review by mykolaharmash — DISMISSED** [review]
To me it looks great ✨ Added just one small suggestion.

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**